src/librustc_mir/hair/pattern/_match.rs

```diff
@@ -740,19 +740,19 @@ impl<'tcx> Constructor<'tcx> {
             }
             ConstantRange(..) => smallvec![self],
             VarLenSlice(self_prefix, self_suffix) => {
-                // A variable-length slice pattern is matched by an infinite collection of fixed-length array
-                // patterns. However it turns out that for each finite set of patterns `P`, all sufficiently
-                // large array lengths are equivalent.
+                // A variable-length slice pattern is matched by an infinite collection of
+                // fixed-length array patterns. However it turns out that for each finite set of
+                // patterns `P`, all sufficiently large array lengths are equivalent.
                 //
                 // Each slice `s` with a "sufficiently-large" length `l ≥ L` that applies
                 // to exactly the subset `Pₜ` of `P` can be transformed to a slice
                 // `sₘ` for each sufficiently-large length `m` that applies to exactly
                 // the same subset of `P`.
                 //
-                // Because of that, each witness for reachability-checking from one of the sufficiently-large
```

Comment:
It would be good to squash these into the commits that introduced them, if possible, to make navigating via git blame more convenient.